terraform/testcases/otlp_grpc_exporter_metric_mock/otconfig.tpl

```diff
@@ -0,0 +1,26 @@
+extensions:
+  pprof:
+    endpoint: 0.0.0.0:1777
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:${grpc_port}
+
+processors:
+  batch/metrics:
+    timeout: 60s
```

Comment:
can we remove the timeout so that all the testcases will use the default batch config